examples/s3/bucket.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: s3.aws.crossplane.io/v1beta1
 kind: Bucket
 metadata:
-  name: test-bucket
+  name: conformancetest2
```

Comment:
Perhaps we should keep `test-bucket` as the `metadata.name` but set a likely-to-be-unique `crossplane.io/external-name` annotation?

Reply:
Yes, sure. As explained above, I just wanted to document how I tested for full transparency. I could not use the "test-bucket" in the tests as bucket names are globally shared with other users. 

One improvement could be to use unique (generated) names for certain resources when provisioning a "conformant testing/example environment" discussed in the above comment. Because we are using shared projects or even some names being globally-scoped, it might be a good idea to generate names not to interfere with each other. I think this is not relevant for examples. Should we have a separate set of manifests/scripts for conformance tests in the future?